app/views/profiles/show.html.erb

```diff
@@ -4,29 +4,65 @@
 
 <div id="profile-tab-root">
   <ul class="nav nav-tabs" role="tablist" id="profile-tabs">
-    <li role="presentation" class="profile-nav active" id="profile-emails-nav"><a href="#profile-emails" data-toggle="tab">Email Delivery</a></li>
+    <li role="presentation" class="profile-nav active" id="profile-emails-nav">
+      <a href="#profile-emails" data-toggle="tab">Preferences</a>
+    </li>
     <% if @profiles_available %>
-      <li role="presentation" class="profile-nav" id="profile-firecloud-nav"><a href="#profile-firecloud" data-toggle="tab">Terra Profile</a></li>
+      <li role="presentation" class="profile-nav" id="profile-firecloud-nav">
+        <a href="#profile-firecloud" data-toggle="tab">Terra Profile</a>
+      </li>
     <% else %>
-      <li role="presentation" class="profile-nav disabled" id="profile-firecloud-nav"><a href="#profile-firecloud" data-toggle="tooltip" title="FireCloud profiles are currently unavailable, please check back later.">Terra Profile</a></li>
+      <li role="presentation" class="profile-nav disabled" id="profile-firecloud-nav">
+        <a href="#profile-firecloud" data-toggle="tooltip"
+           title="FireCloud profiles are currently unavailable, please check back later.">Terra Profile</a>
+      </li>
```

Comment:
With the rebrand having happened a few years ago, I think it's worth updating "FireCloud" to "Terra" when it's in or around code being changed -- especially HTML, CSS, and JS.

```suggestion
      <li role="presentation" class="profile-nav disabled" id="profile-terra-nav">
        <a href="#profile-terra" data-toggle="tooltip"
           title="Terra profiles are currently unavailable, please check back later.">Terra Profile</a>
      </li>
```

Reply:
Ah great catch - totally glossed over all that.